packages/desktop-client/src/components/mobile/accounts/AccountTransactions.jsx

```diff
@@ -0,0 +1,269 @@
+  };
+
+  const onSaveNotes = async (id, notes) => {
+    await send('notes-save', { id, note: notes });
+  };
+
+  const onEditNotes = () => {
+    dispatch(
+      pushModal('notes', {
+        id: account.id,
+        name: account.name,
+        onSave: onSaveNotes,
+      }),
+    );
+  };
+
+  const onCloseAccount = () => {
+    dispatch(openAccountCloseModal(account.id));
+  };
+
+  const onReopenAccount = () => {
+    dispatch(reopenAccount(account.id));
+  };
+
+  const onClick = () => {
+    dispatch(
+      pushModal('account-menu', {
+        accountId: account.id,
+        onSave,
+        onEditNotes,
+        onCloseAccount,
+        onReopenAccount,
+      }),
+    );
+  };
+  return (
+    <View
+      style={{
+        flexDirection: 'row',
+      }}
+    >
+      {account.bankId && (
+        <div
+          style={{
+            margin: 'auto',
+            marginRight: 5,
+            width: 8,
+            height: 8,
+            borderRadius: 8,
+            backgroundColor: pending
+              ? theme.sidebarItemBackgroundPending
+              : failed
+                ? theme.sidebarItemBackgroundFailed
+                : theme.sidebarItemBackgroundPositive,
+            transition: 'transform .3s',
+          }}
+        />
+      )}
+      <Text
+        style={{ ...styles.underlinedText, ...styles.lineClamp(2) }}
+        onClick={onClick}
+      >
+        {`${account.closed ? 'Closed: ' : ''}${account.name}`}
+      </Text>
+    </View>
+  );
+}
+
+const getSchedulesTransform = memoizeOne(id => {
+  const filter = queries.getAccountFilter(id, '_account');
+
+  return q => {
+    q = q.filter({ $and: [filter, { '_account.closed': false }] });
+    return q.orderBy({ next_date: 'desc' });
+  };
+});
+
+function TransactionListWithPreviews({ account }) {
+  const [currentQuery, setCurrentQuery] = useState();
+  const [isSearching, setIsSearching] = useState(false);
+  const [transactions, setTransactions] = useState([]);
+  const prependTransactions = usePreviewTransactions();
+  const allTransactions = useMemo(
+    () =>
+      !isSearching ? prependTransactions.concat(transactions) : transactions,
+    [isSearching, prependTransactions, transactions],
+  );
+
+  const dateFormat = useDateFormat() || 'MM/dd/yyyy';
+  const [_numberFormat] = useLocalPref('numberFormat');
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+
+  const onRefresh = async () => {
+    await dispatch(syncAndDownload(account.id));
+  };
+
+  const makeRootQuery = useCallback(
+    () => queries.makeTransactionsQuery(account.id).options({ splits: 'none' }),
+    [account.id],
+  );
+
+  const paged = useRef(null);
+
+  const updateQuery = useCallback(query => {
+    paged.current?.unsubscribe();
+    paged.current = pagedQuery(
+      query.options({ splits: 'none' }).select('*'),
+      data => setTransactions(data),
+      { pageCount: 10 },
+    );
+  }, []);
+
+  const fetchTransactions = useCallback(async () => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  const fetchTransactions = useCallback(() => {
```